app/graphql/mutations/eras/create_era.rb

```diff
@@ -0,0 +1,17 @@
+class Mutations::Eras::CreateEra < ::Mutations::BaseMutation
+  argument :user_id, ID, required: true
+  argument :name, String, required: false
+  argument :start_date, String, required: false
+  argument :end_date, String, required: false
+  argument :week_number, String, required: false
+  argument :color, String, required: false
+
+  type Types::EraType
+
+  def resolve(attributes)
+    start_date = attributes[:start_date].to_date
+    user_bday = User.where(id: attributes[:user_id])[0].birthdate
+    attributes[:week_number] = ((start_date - user_bday).to_i / 7)
+    Era.create(attributes)
```

Comment:
If we don't accept week_num as an arg (reference from previous comments), we would have to set the week_num differently here

Reply:
Are we going to keep this calculation simple:`/ 7` or do we want to go more in detail?

If we do it this way all weeks will start and end on the weekday on which the user was born. So if the user was born on a Friday all their weeks would start on Fridays. 

I'm not super sure how to manage the alternative so I would be okay with doing it this way for now and creating a ticket for later to improve the quality of this calculation. 

Thoughts?

Reply:
To @leahriffell's point. Even if we don't pass the `week_number` as an argument I think line 14 would still work here. It would just add that key-value pair to the hash and create the era with the week number.

Reply:
I can do some more digging into this. I am ok with leaving it as is for now, maybe add it to our stretch goals. Code freeze will come up faster than we think

Reply:
Maybe we should make this calculation part of a helper module that gets included in the following mutation resolvers:
- create era
- update era
- create event
- update event

Reply:
Yes agreed module sounds great since we'll be using it so many places!

Reply:
Love all these additions. I think that for now I'm find with the / 7 method of doing it. I think a fun refactor would be getting it more precise, however.

and yeah, a Module would be amazing!